### DISCUSSION TITLE
Website memory

### DISCUSSION THREAD

**Discussion by Zer0bytte** [cefsharp/CefSharp#4495] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

Cefsharp 12

### Operating System

Windows 10

### Architecture

x64

### .Net Version

.net z

### Implementation

OffScreen

### Reproduction Steps

There is an website uses much more memory 
is there anyway to reduce memory usage from that website?

### Expected behavior

NA

### Actual behavior

NA

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

**Comment by amaitland** [comment]
In future please use [Discussions](https://github.com/cefsharp/CefSharp/discussions) to ask support related questions.

> is there anyway to reduce memory usage from that website?

If you own the website then use `DevTools` to profile and improve your javascript. Fix memory leaks etc.

To the best of my knowledge the generic only options `CEF` currently provides would reduce functionality e.g. disable image loading, disable javascript, etc

 You might be able to try using some of the `DevTools` protocol methods like https://chromedevtools.github.io/devtools-protocol/tot/Memory/#method-forciblyPurgeJavaScriptMemory

I've not had reason to try this myself so haven't tested the actual behaviour.
